roles/redis/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-_redis_image: redis:7
+_redis_image: redis:latest
```

Comment:
You should revert this change to 7 to avoid bumping redis to a major release if something is release in the future

Reply:
Agreed, I just updated the PR. Thanks!